Cargo.toml

```diff
@@ -13,19 +13,19 @@ publish = false
 autobenches = false
 
 [dependencies]
-aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "2.9.1", default-features = false, features = ["borsh-compat"] }
+aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", branch = "develop", default-features = false, features = ["borsh-compat"] }
```

Comment:
Why did you decide to use a branch here instead of a tag?

Reply:
Why not? For the most latest basically.

Reply:
Overall because the develop could potentially contain breaking changes that the released version couldn't. But probably not for this case.